contracts/core/ModuleCore.sol

```diff
@@ -195,7 +199,10 @@ contract ModuleCore is OwnableUpgradeable, UUPSUpgradeable, PsmCore, Initialize,
      * @notice update value of PSMBaseRedemption fees
      * @param newPsmBaseRedemptionFeePrecentage new value of fees
      */
-    function updatePsmBaseRedemptionFeePrecentage(uint256 newPsmBaseRedemptionFeePrecentage) external onlyConfig {
+    function updatePsmBaseRedemptionFeePrecentage(Id id, uint256 newPsmBaseRedemptionFeePrecentage)
+        external
+        onlyConfig
+    {
```

Comment:
this function still updates the global base redemption fee. not the individual Id base redemption rate. Also don't forget to make 1 function that calls this in the config contract